sphinx/writers/latex.py

```diff
@@ -894,8 +886,6 @@ def visit_document(self, node):
         self.footnotestack.append(self.collect_footnotes(node))
         self.curfilestack.append(node.get('docname', ''))
         if self.first_document == 1:
-            # the first document is all the regular content ...
```

Comment:
I'll remove this, and try to create a test case, but on my I side, I use this only for multiple latex documents, and this is fine.